lya/data.py

```diff
@@ -22,7 +23,11 @@ def load_lya_data(filename=lya_data_path):
         K, matter power spectrum, and p(k) uncertainty arrays,
         encapsulated in Data object
     """
+    x, y, y_unc = np.loadtxt(filename, unpack=True)
+
+    data = Data(x, y, y_unc)
```

Comment:
won't this cause infinite recursion?

Reply:
We also need to set the `.x`, `y`, `y_unc` attributes

Reply:
No I just tested it and it works fine. 

you just run 

`data = load_lya_data(...)`

and then you can call `data.x` `data.y` etc. 

Is this ok?

Reply:
Oh sorry- I was misreading the code on github and thought this code was under `__init__`!

Reply:
Yes, sorry!  (I missed the fact that viewing the code on github was hiding lines!)